Userland/Libraries/LibSoftGPU/Device.cpp

```diff
@@ -1004,4 +1004,8 @@ void Device::set_sampler_config(unsigned sampler, SamplerConfig const& config)
     m_samplers[sampler].set_config(config);
 }
 
+void Device::set_light_state(unsigned int light_id, Light const& light)
+{
+    m_lights.at(light_id) = light;
+}
```

Comment:
Missing newline between function and namespace

Reply:
`clang-format` is doing this.